package.json

```diff
@@ -14,7 +14,8 @@
     "lint": "eslint .",
     "migrations": "sequelize db:migrate",
     "reset-database": "npx redis-cli FLUSHALL && sequelize db:drop && sequelize db:create",
-    "docs": "./node_modules/.bin/jsdoc -c jsdoc.json"
+    "docs": "./node_modules/.bin/jsdoc -c jsdoc.json",
+    "delete-partners": "babel-node ./server/deletePartners.js"
```

Comment:
IS this command be `delete-partners` or `delete-invalid-partners` as it might delete all rows

Reply:
Yes, it should delete all rows since they all do not have location field currently